conf/script/ha_setup

```diff
@@ -200,6 +201,23 @@ config_decision_maker() {
 
     cp -rf /tmp/decision_monitor_conf.json /etc/cortx/ha/decision_monitor_conf.json
     rm -rf /tmp/decision_monitor_conf.json
+
+    # Configure ha_iem event get info from Decision maker
+    config_ha_iem_event ${host1} ${uuid1} ${host2} ${uuid2}
+}
+
+config_ha_iem_event() {
+    echo "Configure IEM for ha event"
+    cp -rf ${SOURCE_HA_CONF}/iem_ha.json ${HA_CONF}
+
+    sed -i -e "s|<HOST1>|$1|g" \
+        -e "s|<HOST2>|$3|g" \
+        -e "s|<UUID1>|$2|g" \
+        -e "s|<UUID2>|$4|g" ${HA_CONF}/iem_ha.json
+
+    # Configure alert for HA IEM
+    setfacl -R -m u:hacluster:rwx ${HA_LOG}
```

Comment:
Check the return code for these two commands.

Reply:
Done